beacon-chain/blockchain/service.go

```diff
@@ -114,7 +114,6 @@ func NewService(ctx context.Context, cfg *Config) (*Service, error) {
 		maxRoutines:       cfg.MaxRoutines,
 		stateNotifier:     cfg.StateNotifier,
 		forkChoiceStore:   cfg.ForkChoiceStore,
-		initSyncState:     make(map[[32]byte]*stateTrie.BeaconState),
```

Comment:
@farazdagi this was only used in ClearCachedStates and in beacon-chain/blockchain/process_block_test.go. The map is never read, only written to.

Reply:
- I haven't introduced that variable into service struct, so have little idea why it is only written into and never read from.
- @terencechain can you please double-check that we are not removing something that should be extended/used?

Reply:
This was used a while back to cache states during init sync. It can be deleted now